api/v1alpha1/database_types.go

```diff
@@ -214,6 +214,7 @@ type ProfileResponse struct {
 	EngineType      string `json:"engineType"`
 	LatestVersionId string `json:"latestVersionId"`
 	Topology        string `json:"topology"`
+	SystemProfile   bool   `json:"systemProfile"`
```

Comment:
Also going to add "Status" here in the ProfileResponse struct
Cc @manavrajvanshi

Reply:
That would be a really nice thing to have, thanks for figuring out SystemProfile flag and Status flags! 💯 💯